zephyr/src/indigo_api_callback_dut.c

```diff
@@ -347,7 +347,8 @@ static int generate_hostapd_config(char *output, int output_size, struct packet_
             token = strtok(buffer, delimit);
 
             while(token != NULL) {
-                sprintf(cfg_item, "%s=%s\n", cfg->config_name, token);
+                ret = snprintf(cfg_item, sizeof(cfg_item), "%s=%s\n", cfg->config_name, token);
+		CHECK_RET(ret);
```

Comment:
you can combine these two: 

```suggestion
CHECK_SPRINTF(cfg_item, sizeof(cfg_item), "%s=%s\n", cfg->config_name, token);
```